CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.0.3](https://github.com/rhobs/monitoring-stack-operator/commit/) (2021-11-04)
+
+
+### Features
+
+* add static manifests for the grafana-operator ([e0040b6](https://github.com/rhobs/monitoring-stack-operator/commit/e0040b6c9ef50d0ff777962e09bde8118a054a1d))
+* add thanos querier CRD ([#52](https://github.com/rhobs/monitoring-stack-operator/issues/52)) ([0dd9499](https://github.com/rhobs/monitoring-stack-operator/commit/0dd94995b006c4df8b13326ae8ab8a9831eb23fc))
+* deploy an instance of the grafana operator ([409a95e](https://github.com/rhobs/monitoring-stack-operator/commit/409a95e986b8f2a3151e327c20c6c2ae5c83b863))
+* implement self-scraping for monitoring stacks ([632f913](https://github.com/rhobs/monitoring-stack-operator/commit/632f9133ae333bae49cbc33912c6d9093d533a24))
+* MonitoringStack controller that deploys Prometheus ([#40](https://github.com/rhobs/monitoring-stack-operator/issues/40)) ([f16a977](https://github.com/rhobs/monitoring-stack-operator/commit/f16a9772add878df90b37fc7cf2bd95f26ce94f3))
```

Comment:
Should we merge these two?

Reply:
I don't feel too strongly about that .. Seems like to different features to me.

Reply:
Ok we can leave them separate. Can we just unify the casing to either use uppercase or lowercase everywhere when starting a sentence?